packages/sei-cosmwasm/src/msg.rs

```diff
@@ -24,7 +22,7 @@ pub enum SeiMsg {
         contract_address: Addr,
     },
     CancelOrders {
-        order_ids: Vec<u64>,
+        cancellations: Vec<Cancellation>,
         contract_address: Addr,
```

Comment:
do we need `contract_address` if the cancellation type is containing a contract address already?

Reply:
Yea it is needed here: https://github.com/sei-protocol/sei-chain/blob/main/x/dex/client/wasm/encoder.go#L33 . As for the need for contract address in the `Cancellation` type, I was matching Cancellation type 1-1 with the type in sei-chain.